src/modules/kubernetes.rs

```diff
@@ -239,6 +280,26 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     Some(module)
 }
 
+fn parse_kubeconfigs<I>(raw_kubeconfigs: I) -> Vec<Document>
+where
+    I: Iterator<Item = Option<String>>,
+{
+    raw_kubeconfigs
+        .filter_map(|content| match content {
+            Some(value) => match value.chars().next() {
+                // Parsing as json is about an order of magnitude faster than parsing
+                // as yaml, so do that if possible.
+                Some('{') => serde_json::from_str(&value).ok().map(Document::Json),
```

Comment:
I would prefer to fall back to `YamlLoader::load_from_str` if `serde_json::from_str` fails, even if the first character is `{`.

Reply:
Added a fallback and test, ptal